src/components/ui/AnswerBoard.js

```diff
@@ -136,21 +131,25 @@ class AnswerBoard extends React.Component {
 
   };
 
-  onButtonClick = (index) => {
-
-  };
-
   render() {
-    const {questionNumber, userAnswer, language} = this.props;
-    const question = questions[questionNumber-1];
-    const originalAnswer = answers[questionNumber-1].Answer;
+    const {language} = this.props;
+    const {Id, Selected} = this.props.location.state;
+    const questionNumber = Id.toString();
+    const userAnswer = Selected.toString();
+    const question = questions[questionNumber - 1];
+    const originalAnswer = answers[questionNumber - 1].Answer;
     const correct = (userAnswer === originalAnswer);
 
     return (
```

Comment:
Can you make below here 4 spaces indentation??

Reply:
Done! Thx